src/sage/matroids/matroid.pyx

```diff
@@ -8028,109 +8028,58 @@ cdef class Matroid(SageObject):
 
         return [F for F in FF if fsol[F]]
 
-    def chow_ring(self, R=None):
+    def chow_ring(self, R, augmented=False, presentation=None):
         r"""
-        Return the Chow ring of ``self`` over ``R``.
+        Return the (augmented) Chow ring of ``self`` over ``R``.
 
-        Let `M` be a matroid and `R` be a commutative ring.
-        The *Chow ring* of `M` is the quotient ring
-
-        .. MATH::
-
-            A^*(M)_R := R[x_{F_1}, \ldots, x_{F_k}] / (Q_M + L_M),
-
-        where
-
-        - `F_1, \ldots, F_k` are the non-empty proper flats of `M`,
-        - `Q_M` is the ideal generated by all `x_{F_i} x_{F_j}` where
-          `F_i` and `F_j` are incomparable elements in the lattice of
-          flats, and
-        - `L_M` is the ideal generated by all linear forms
-
-          .. MATH::
-
-              \sum_{i_1 \in F} x_F - \sum_{i_2 \in F} x_F
+        .. SEEALSO::
 
-          for all `i_1 \neq i_2 \in E`.
+            - :mod:`sage.matroids.chow_ring_ideal`
+            - :mod:`sage.matroids.chow_ring`
 
         INPUT:
 
-        - ``R`` -- (default: `\ZZ`) the base ring
+        - ``M`` -- matroid
+        - ``R`` -- commutative ring
+        - ``augmented`` -- boolean; when ``True``, this is the augmented
+            Chow ring and if ``False``, this is the non-augmented Chow ring
+        - ``presentation`` -- string (default: ``None``); one of the following
+            (ignored if ``augmented=False``)
+
+            * ``"fy"`` - the Feitchner-Yuzvinsky presentation
+            * ``"atom-free"`` - the atom-free presentation
 
         EXAMPLES::
 
             sage: M = matroids.Wheel(2)
-            sage: A = M.chow_ring(); A
-            Chow ring of Wheel(2): Regular matroid of rank 2 on 4 elements
-             with 5 bases over Integer Ring
-            sage: A.gens()                                                              # needs sage.libs.singular
-            (A23, A23, A23)
-            sage: A23 = A.gen(0)                                                        # needs sage.libs.singular
-            sage: A23*A23                                                               # needs sage.libs.singular
+            sage: A = M.chow_ring(R=ZZ, augmented=False); A
+            Chow ring of Wheel(2): Regular matroid of rank 2 on 4 elements with 5 bases
+            sage: A.defining_ideal()._gens_constructor(A.defining_ideal().ring())
+            [A0*A1, A0*A23, A1*A23, A0 + A0123, A1 + A0123, A23 + A0123]
+            sage: A23 = A.gen(0)
+            sage: A23*A23
             0
 
         We construct a more interesting example using the Fano matroid::
 
             sage: M = matroids.catalog.Fano()
-            sage: A = M.chow_ring(QQ); A
-            Chow ring of Fano: Binary matroid of rank 3 on 7 elements,
-             type (3, 0) over Rational Field
-
-        Next we get the non-trivial generators and do some computations::
```

Comment:
Can this multiplication table be added back in? It should give the same result.

Reply:
I don't remember what the computation in the example was. Could you go through it again?